### ISSUE TITLE
DefaultCupertinoDatePicker should support internationalization

### ISSUE THREAD

**Issue by blattr** [stryder-dev/flutter_platform_widgets#338]
First of all, I love this plugin and I hope, that more widgets are coming up. 

I tried to override the default title of the "done" and "cancel" button, but it seems that there is no possibility to do so. Am I wrong or might be the solution to make doneLabel and cancelLabel part of the **CupertinoDatePickerData** and use it like **data?.doneLabel** and **data?.cancelLabel**?

```dart
@override
Widget build(BuildContext context) {
  return Container(
    height: modalHeight,
    color: modalColor ?? Theme.of(context).canvasColor,
    child: Stack(
      children: [
        CupertinoDatePicker(
          key: data?.key,
          mode: data?.mode ?? mode,
          onDateTimeChanged: (value) {
            data?.onDateTimeChanged?.call(value);
            onDateTimeChanged.call(value);
          },
          initialDateTime: contentData.initialDate,
          minimumDate: contentData.firstDate,
          maximumDate: contentData.lastDate,
          backgroundColor: data?.backgroundColor,
          dateOrder: data?.dateOrder,
          maximumYear: data?.maximumYear,
          minimumYear: data?.minimumYear ?? 1,
          minuteInterval: data?.minuteInterval ?? 1,
          use24hFormat: data?.use24hFormat ?? false,
        ),
        Row(
          children: [
            PlatformTextButton(
              onPressed: () {
                Navigator.pop(context);
              },
              child: Text(cancelLabel ?? 'Cancel'),
            ),
            const Spacer(),
            PlatformTextButton(
              onPressed: () {
                Navigator.pop(context, contentData.selectedDate);
              },
              child: Text(doneLabel ?? 'Done'),
            ),
          ],
        ),
      ],
    ),
  );
```

**Comment by aqwert** [comment]
I can see the value in that for when using the default `DefaultCupertinoDatePicker`. Will look into adding that soon.

(Sorry for the delay, I was on holiday break)

**Comment by aqwert** [comment]
Just published `1.12.1` that fixes this. Thanks
